extensions/custom-footer/src/Footer.jsx

```diff
@@ -0,0 +1,43 @@
+import {
+  Icon,
+  InlineLayout,
+  InlineStack,
+  Link,
+  Text,
+  reactExtension,
+  useShop,
+} from '@shopify/ui-extensions-react/checkout';
+
+export const checkoutFooter = reactExtension(
+  'purchase.checkout.footer.render-after',
+  () => <Extension />,
+);
+
+export const thankYouFooter = reactExtension(
+  'purchase.thank-you.footer.render-after',
+  () => <Extension />,
+);
+
+function Extension() {
```

Comment:
Can you share an image of what this extension looks like in the checkout?

Reply:
This is how the footer will look, I'll add to the PR's description too.

![image](https://github.com/Shopify/example-checkout--custom-header-footer--react/assets/1685986/8f802e42-c1d4-403b-8290-bf2719d3638a)

Reply:
Thank you!